.gitignore

```diff
@@ -6,3 +6,4 @@ node_modules
 
 **/tsconfig.tsbuildinfo
 .DS_Store
+.idea
```

Comment:
I added this because I'm a dirty RubyMine user and it kept picking up changes to this RubyMine file.

Reply:
😂